docs/rfcs/README.md

```diff
@@ -46,15 +46,19 @@ We use a [Lazy Consensus](https://www.apache.org/foundation/glossary.html#LazyCo
 1. *Quorum*: For an RFC to be mergeable, it needs to have at least **two approvals** from the
    approvers set as well as approvals from any additional stakeholders.
 2. *Waiting period*: Maintainers need to announce their intent to merge the RFC with a GitHub
-   comment. They will need to add a `rfc:final-comment-period` label to the PR and wait for at least
-   **4 business days** after making the announcement to merge the RFC.
+   comment. They will need to add a `rfc:final-comment-period` label to the PR, comment on the PR
+   and note the final comment period in the #otel-collector-dev CNCF Slack channel, and wait for at
+   least **4 business days** after making the announcement to merge the RFC.
```

Comment:
I worry the idea of business days might be fuzzy considering the amount of approvers/maintainers in different geos.

Would it make more sense to say 5 days (this would account for plus or minus a day depending on which part of the world you're in)?

Reply:
> I worry the idea of business days might be fuzzy considering the amount of approvers/maintainers in different geos.
> Would it make more sense to say 5 days (this would account for plus or minus a day depending on which part of the world you're in)?

Since this is already part of the text and was already approved at #11416, I think we can discuss this independently from this PR, see #11763